repose-aggregator/components/filters/simple-rbac-filter/src/main/scala/org/openrepose/filters/simplerbac/SimpleRbacFilter.scala

```diff
@@ -225,12 +225,34 @@ class SimpleRbacFilter @Inject()(configurationService: ConfigurationService,
     }
 
     parsed.flatMap { values =>
-        val header = s"""<application xmlns:rax="http://docs.rackspace.com/api"
-                        |         xmlns:xsd="http://www.w3.org/2001/XMLSchema"
-                        |         xmlns="http://wadl.dev.java.net/2009/02"
-                        |    >
-                        |  <resources base="http://localhost">""".stripMargin
-        val resources = values.map { value =>
+        val header = s"""<application xmlns="http://wadl.dev.java.net/2009/02"
+                        |             xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+                        |             xmlns:rax="http://docs.rackspace.com/api"
+                        |>
+                        |  <resources base="http://localhost">
+                        |""".stripMargin
+        val tenants = Option(configuration.getTenantsHeaderName) match {
+          case Some(tenantsHeader) =>
+            s""">
+               |        <request>
+               |          <param name="$tenantsHeader"
+               |                 style="header"
+               |                 required="true"
+               |                 type="xsd:int"
+               |                 repeating="true"
+               |                 rax:isTenant="true"/>
+               |          <param name="$tenantsHeader"
```

Comment:
I'm confused, this seems to be a repeat other than changing the type to boolean?

Reply:
Yes. It says the header can contain either an `int` or a `boolean` value. This is actually what caused the error to appear in the API-Checker tests. It hit a corner case where the case of the `name` attribute was different and caused weird things to happen. There are now tests in API-Checker and this proves as long as they are the same case that it still works.

Reply:
when would it have a boolean?

Reply:
It isn't that it should, it is that it is legal by the WADL standard so it proves we don't blow up.

Reply:
We are validating that API-Checker will handle a valid WADL. It is valid in a WADL to define that a particular header will have a value of either type x or type y. The `rax:isTenant` extension just enforces this capability. I could have used a non-standard type like Positive Integers and/or UUID's. The `int`'s and `boolean`'s happen to be standard types that I didn't have to define.

Reply:
On top of that, that is the corner case scenario that caused the bug in API-Checker to surface.

Reply:
this is the prod code that generates the wadl we feed to api checker right? why would we ever say a boolean is a valid value in our wadl. although now you bring up an excellent point that int is too restrictive. we can totally have tenant id's with other values, and in fact do have a bunch `hybrid:...` and there will be more like that in the future.

Reply:
Crap. I had the wrong context in my head.

Reply:
You're correct. This should only need the one. Do we make it a `String`?